### PR TITLE
Use distri-example in install_publish tests

### PR DESCRIPTION
**Pull request by mimi1vx**
https://progress.opensuse.org/issues/127622

For test which uses full distri-opensuse set 'FULL_OPENSUSE_TEST'

http://quasar.suse.cz/tests/2768

### CODE REVIEW

**Comment by okurz** [comment]
looks very nice. Just needs a typo fix s/disri/distri/

File: tests/install/test_distribution.pm [hunk 1]
```diff
@@ -6,9 +6,11 @@ use utils;
 sub run {
     return 1 if check_var('OPENQA_FROM_GIT', 1);
     diag('assuming to be in terminal');
-    diag('initialize working copy of openSUSE tests distribution with correct user');
-    assert_script_run('retry -s 30 -- sh -c "username=bernhard email=bernhard@susetest /usr/share/openqa/script/fetchneedles"', 3600);
-    save_screenshot;
+    if (get_var('FULL_OPENSUSE_TEST', 0)) {
```
Comment (perlpunk) [inline tests/install/test_distribution.pm:9]:
```suggestion
    if (get_var('FULL_OPENSUSE_TEST')) {
```

**Review by perlpunk — APPROVED** [review]

**Comment by mimi1vx** [comment]
@okurz same thing in https://openqa.opensuse.org/tests/3341724#step/start_test/5 works ...

**Comment by perlpunk** [comment]
However, I saw the same error yesterday here, only for one of the tests.

Regarding what code the test uses - the triggered test on openqa is using code from the PR, only the scenarios file is used from master.

**Pull request merged**